src/corehost/build.proj

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
for *nix or for all OS?

I chose using msbuild to centralize parsing arguments and avoid duplicating logic in script files. otherwise we still need the same logic in build.sh and a "build-native.sh" to parse/generate a TargetRid